go/disk.go

```diff
@@ -208,8 +208,8 @@ func (d *RawDisk) create() error {
 	return f.Truncate(int64(d.Size) * mib)
 }
 
-// The current disk size in MiB.
-func (d *RawDisk) getCurrentSize() (int, error) {
+// GetCurrentSize retursn the current disk size in MiB.
```

Comment:
retursn -> returns

also below

Reply:
Oops, sorry about that, I fixed it. Thank!